pints/_log_pdfs.py

```diff
@@ -115,6 +115,177 @@ def sample(self, n=1):
         raise NotImplementedError
 
 
+class PooledLogPDF(LogPDF):
+    r"""
+    Calculates a sum of :class:`LogPDF` objects, while pooling selected
```

Comment:
Creates an object of :class:`LogPDF`, with a log-pdf which is the sum of the individual log-pdfs.

Reply:
Done